### PR TITLE
Include `shuffles` in `PinnedConstraintSystem::fmt`

### PR DESCRIPTION
**Pull request by therealyingtong**

### CODE REVIEW

**Review by han0110 — APPROVED** [review]
LGTM! Nice catch.

(the clippy error should be fixed after #192 is merged)

**Comment by han0110** [comment]
Just realized the change might lead to different `transcript_repr` even the circuit doesn't use `shuffle`, so I think to keep backward compatability we need to skip the `shuffle` field when it's empty.

File: halo2_proofs/src/plonk/circuit.rs [hunk 1]
```diff
@@ -1623,6 +1623,7 @@ impl<'a, F: Field> std::fmt::Debug for PinnedConstraintSystem<'a, F> {
             .field("fixed_queries", self.fixed_queries)
             .field("permutation", self.permutation)
             .field("lookups", self.lookups)
+            .field("shuffles", self.shuffles)
```
Comment (han0110) [inline halo2_proofs/src/plonk/circuit.rs:1626]:
```suggestion
            .field("lookups", self.lookups);
        if !self.shuffles.is_empty() {
            debug_struct.field("shuffles", self.shuffles);
        }
        debug_struct
```

**Review by han0110 — CHANGES_REQUESTED** [review]

**Review by CPerezz — APPROVED** [review]
LGTM!

**Comment by CPerezz** [comment]
Closing as @han0110 already cherry-picked this in  #226

**Pull request closed without merging**